### ISSUE TITLE
[Bug Report] New video.js player does not scale properly in Firefox.

### ISSUE THREAD

**Issue by stashTX** [stashapp/stash#2513]
**Describe the bug**
When using firefox (99.0.1 64bit) in windows 11,  the video player seems to scale improperly.  It makes the web page super wide and there's always a horizontal scroll bar.   This issue does not appear in Chrome on the same machine.

I've disabled all my plugins and tried clearing cache and using private browsing mode.  Page always renders this way.

**To Reproduce**
1. Load any scene page in Firefox on Windows 11.
2. Observe page rendering

**Expected behavior**
Video window scales properly.

**Screenshots**

![stash_bug](https://user-images.githubusercontent.com/91716824/163683840-118cd053-ac48-4a96-ba26-181c7e5d6502.png)

**Stash Version: (from Settings -> About):**
v0.14.0

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Firefox
 - Version: 99.0.1

**Comment by WithoutPants** [comment]
This looks like a duplicate of #2510 which is fixed in the development branch. Can you check if the following custom css fixes the issue for you:

```
.VideoPlayer .vjs-touch-enabled {
    margin: 0px;
    width: 100%;
}
```

**Comment by WithoutPants** [comment]
Closing as presumably fixed in 0.15. Please let me know if it is not.
